pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -170,6 +170,17 @@ type ScorePlugin interface {
 	Score(pc *PluginContext, p *v1.Pod, nodeName string) (int, *Status)
 }
 
+// NormalizeScorePlugin is an interface that must be implemented by "normalize score"
+// plugins to normalize the node scoring results produced by the same plugin's "Score"
+// method.
+type NormalizeScorePlugin interface {
```

Comment:
If each plugin that implements `NormalizeScore` should also have a `Score` method, why not embed ScorePlugin into NormalizeScorePlugin like this?

```go
type NormalizeScorePlugin interface {
	ScorePlugin

	NormalizeScore(pc *PluginContext, scores NodeScoreList) *Status
}
```

Reply:
Sounds like a good idea.